source/Octopus.Tentacle.CommonTestUtils/Builders/StartScriptCommandV3AlphaBuilder.cs

```diff
@@ -1,22 +1,24 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using Octopus.Tentacle.Contracts;
+using Octopus.Tentacle.Contracts.Builders;
 using Octopus.Tentacle.Contracts.ScriptServiceV3Alpha;
 
-namespace Octopus.Tentacle.Contracts.Builders
+namespace Octopus.Tentacle.CommonTestUtils.Builders
 {
     public class StartScriptCommandV3AlphaBuilder
     {
         readonly List<ScriptFile> files = new();
         readonly List<string> arguments = new();
         readonly Dictionary<ScriptType, string> additionalScripts = new();
         StringBuilder scriptBody = new(string.Empty);
-        ScriptIsolationLevel isolation = ScriptIsolationLevel.FullIsolation;
+        ScriptIsolationLevel isolation = ScriptIsolationLevel.NoIsolation;
```

Comment:
Why did this change? I would have thought that `FullIsolation` would be a safe default. Just after some clarification, thanks.

Reply:
So chasing this through, It looks like Nathan changed this in #764 as he wanted to use the builders in [OctopusDeploy/DynamicWorkers](https://github.com/OctopusDeploy/DynamicWorkers/pull/683)

They were previous to that change had no isolation, but then were changed to full isolation as a default for the now-public builders.

I've followed up with CloudPT to let them know dynamic workers will need updating with the new contract changes in #845